### DISCUSSION TITLE
✨ Xcode 15.1

### DISCUSSION THREAD

**Discussion by helinanever** [codemagic-ci-cd/codemagic-docs#2163] [Announcements]
We have released a new build machine image with **Xcode 15.1 Beta 2** available. This image comes with macOS Sonoma 14.0 and has all the latest SDKs and runtimes including visionOS installed. You can use Xcode 15.1 by specifying `15.1` or `edge` as the Xcode version in your build configuration.

See more details about the build machine specs and preinstalled software in our [documentation](https://docs.codemagic.io/specs/versions-macos/).

**Comment by SGXiaoLin** [comment]
Hello, the package of Xcode 15.0.1 version cannot be submitted for review. Can you open the specified 15.0 version?

**Reply by helinanever** [reply]
Hi @SGXiaoLin, Apple resolved this issue a few days ago: https://developer.apple.com/forums/thread/740853?answerId=771131022#771131022
Are you still experiencing issues with submitting for review?

**Comment by helinanever** [comment]
The Xcode 15.1 image now has **Beta 3** available.

**Comment by helinanever** [comment]
We've rolled out the **release** version of Xcode 15.1. The 15.1 image now comes with macOS Sonoma 14.2 and has all the runtimes updated to stable versions.
